synapse/storage/profile.py

```diff
@@ -71,8 +71,6 @@ def get_from_remote_profile_cache(self, user_id):
             desc="get_from_remote_profile_cache",
         )
 
-
-class ProfileStore(ProfileWorkerStore):
```

Comment:
Is there not a danger of races if the `*_remote_profile_cache` methods are called from a worker?

Reply:
on closer inspection, it looks like efforts are made to *not* call those methods from a worker, in which case I think they should probably stay in ProfileStore rather than moving.

Reply:
Woops, fixed